home/modules/ROOT/pages/index.adoc

```diff
@@ -8,10 +8,17 @@
 We're no longer updating the content on this site regularly. Check out the https://www.couchbase.com/support-policy[Couchbase Support Policy] for information on version support and End of Life schedule.
 
 ifdef::primary-site-url[]
-Click {primary-site-url}[here] to return to the main documentation site.
+This page contains links to End-of-Life, archived documentation, for reference purposes, as well as links to docs for current, GA products. Click {primary-site-url}[here] to return to the main documentation site.
```

Comment:
While you're refactoring this:

* ventilated prose
* opposrtunity to get rid of "Click" by rewording as `Alternatively, return to {url}[the main documentation site]` ?

Reply:
Thanks - yes, didn't ventilated prose all of the inherited content.
Will change that now, and get rid of Click.  :+1: